src/System.IO.FileSystem/src/System/IO/FileSystem.Attributes.Windows.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Did any code in this file change, or is it just moved without modification from the original source file?

Reply:
Just moved, no changes.